src/display/annotation_helper.js

```diff
@@ -273,6 +361,9 @@ var AnnotationUtils = (function AnnotationUtilsClosure() {
   }
 
   function getHtmlElement(data, objs) {
+    if (!data.hasHtml) {
```

Comment:
This change should not be necessary. It is already handled in https://github.com/mozilla/pdf.js/blob/master/src/display/annotation_layer.js#L330-L332 just before calling this method.